rspirv/sr/autogen_ops.rs

```diff
@@ -18,9 +18,31 @@
 
 use crate::sr::{Token, Type};
 #[derive(Clone, Debug, Eq, PartialEq)]
+pub enum Branch {
+    Branch {
+        target_label: spirv::Word,
```

Comment:
So `spirv:Word` is the DR's way to "link" instructions. Here I think target_label should be `Token<BasicBlock>`.